src/guiguts/search.py

```diff
@@ -57,9 +57,9 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         except AttributeError:
             SearchDialog.selection = tk.BooleanVar(value=False)
 
-        kwargs["resize_y"] = False
+        # kwargs["resize_y"] = False
```

Comment:
🤣  When I uncomment line 60, the search box looks like this:

<img width="182" alt="image" src="https://github.com/user-attachments/assets/a9dcfae9-591d-4481-98a7-3d6063a36ecf">

It  flashes larger, then reduces to the thin line you see. It's resizeable -- just hover over the line and you can drag it open.

---

Reply:
Was that with `--nohome` or possibly using a setting from the Prefs file that is not valid now it's resizable?